rules_conditions.md

```diff
@@ -0,0 +1,226 @@
+    Operator: ">"
+    Value: 300
+    Datatype: int
+```
+
+In this scenario, the rule applies to traces with more than 300 spans.
+For details about all supported special fields, check out [documentation here](https://docs.honeycomb.io/manage-data-volume/refinery/configuration/#refinery-telemetry)
+
+### Virtual Fields
+
+To handle specific scenarios when rules are evaluated before the arrival of root spans, Refinery introduces the concept of virtual fields. These fields provide metadata about traces that have timed out while waiting for their root span.
+
+```yaml
+Rules:
+    - Name: Drop any big traces
+      Drop: true
+      Conditions:
+        Field: "?.NUM_DESCENDANTS"
+        Operator: ">"
+        Value: 1000
+        Datatype: int
+
+```
+
+This example shows a rule that drops traces containing more than 1000 spans, using the virtual field `?.NUM_DESCENDANTS`.
+
+#### Supported Virtual Fields
+
+All virtual fields will be prefixed with `?.` to distinguish them from normal fields.
+Currently only one virtual field is supported.
+
+- `?.NUM_DESCENDANTS`: the current number of child elements contained within a trace.
+
+## `Fields`
+
+The `Fields` parameter is exactly equivalent to `Field`, except that it must be expressed as an array of strings.
+The array defines a sequences of `Field` names that are checked in order for each span being considered.
+The first field that `exists` on any given span is used for the condition.
+
+### Example use of `Fields`
+
+This example shows how one might write a rule designed to cope with an expected name change of a key field.
+
+```yaml
+Conditions:
+    Fields:
+        - http.status
+        - http.request.status
+    Operator: =
+    Value: 200
+    Datatype: int
+```
+
+## `Operator`
+
+The `Operator` parameter controls how rules are evaluated.
+Because YAML treats certain characters like less-than (`<`) specially, it is good practice to always enclose the basic comparison operators in single quotes (like `'<'`).
+
+The `Operator` may be one of the following:
+
+### `'='`
+
+Basic comparison -- `equals`.
+True if the value of the named Field is equal to the `Value` specified.
+See [`Datatype`](#datatype) ffor how different datatypes are handled.
+
+### `'!='`
+
+Basic comparison -- `not equals`.
+True if the value of the named Field is not equal to the `Value` specified.
+See [`Datatype`](#datatype) ffor how different datatypes are handled.
+
+### `'<'`
+
+Basic comparison -- `less than`.
+True if the value of the named Field is less than the `Value` specified.
+See [`Datatype`](#datatype) ffor how different datatypes are handled.
+
+### `'<='`
+
+Basic comparison -- `less than or equal to`.
+True if the value of the named Field is less than or equal to the `Value` specified.
+See [`Datatype`](#datatype) ffor how different datatypes are handled.
+
+### `'>'`
+
+Basic comparison -- `greater than`.
+True if the value of the named Field is greater than the `Value` specified.
```

Comment:
```suggestion
Returns `true` if the value of the named Field is greater than the `Value` specified.
```

Reply:
I changed "Returns true" to "The result is true" because "returns" isn't really meaningful here. OK?

Reply:
That works! Thanks, Kent!